packages/fxa-settings/src/components/BentoMenu/index.test.tsx

```diff
@@ -11,21 +11,23 @@ describe('BentoMenu', () => {
     render(<BentoMenu />);
 
     const toggleButton = screen.getByTestId('drop-down-bento-menu-toggle');
-    const dropDownId = 'drop-down-bento-menu';
-    const dropDown = screen.queryByTestId(dropDownId);
+    const dropDown = screen.queryByTestId('drop-down-bento-menu');
 
-    expect(toggleButton).toHaveAttribute('title', 'bento-menu-title');
-    expect(toggleButton).toHaveAttribute('aria-controls', dropDownId);
+    expect(toggleButton).toHaveAttribute('title', 'Firefox Bento Menu');
+    expect(toggleButton).toHaveAttribute(
+      'aria-controls',
+      'drop-down-bento-menu'
+    );
     expect(toggleButton).toHaveAttribute('aria-expanded', 'false');
-    expect(dropDown).not.toBeInTheDocument;
+    expect(dropDown).not.toBeInTheDocument();
 
     fireEvent.click(toggleButton);
     expect(toggleButton).toHaveAttribute('aria-expanded', 'true');
     expect(dropDown).toBeInTheDocument;
```

Comment:
I spent a few minutes trying to get some of these `.toBeInTheDocument` tests changed to `.toBeInTheDocument()` but I don't know why they're failing. I'll file a follow up issue to check all of our `toBeInTheDocument` and `toBeTruthy` calls across React tests because we have a few for some reason.

Reply:
follow up: #12868